pkg/storagev2/storagetest/transaction.go

```diff
@@ -336,7 +336,7 @@ func checkTxChunkStoreFinishedTxInvariants(t *testing.T, store storage.TxChunkSt
 
 	ctx := context.Background()
 	want := storage.ErrTxDone
-	o007 := swarm.NewChunk(swarm.NewAddress([]byte("007")), []byte("Hello, World!"))
+	o007 := chunktest.GenerateTestRandomChunk()
```

Comment:
I'd suggest not using these `GenerateTestRandomChunk' calls here, as it makes it harder to debug the values when tests fail. The same goes for the whole file.